pkg/metrics/store/sqlite.go

```diff
@@ -117,25 +126,26 @@ func insert(ctx context.Context, dbRW *sql.DB, table string, ms ...pkgmetrics.Me
 
 	// Build the query with placeholders for all metrics
 	query := fmt.Sprintf(
-		"INSERT OR REPLACE INTO %s (%s, %s, %s, %s, %s) VALUES ",
+		"INSERT OR REPLACE INTO %s (%s, %s, %s, %s, %s, %s) VALUES ",
 		table,
 		ColumnUnixMilliseconds,
 		ColumnComponentName,
 		ColumnMetricName,
-		ColumnMetricLabel,
 		ColumnMetricValue,
+		ColumnMetricLabelName,
+		ColumnMetricLabelValue,
```

Comment:
I have always had a question that how we store these metrics if there are many label keys and values for the metrics.

Reply:
@eahydra  Our assumption is, we only allow one custom/secondary label (e.g., GPU UUID, mount point) to restrict the number of columns, and also support SQL selects based on the GPU UUID or mount points.

Reply:
Hmm.. I don't think the assumption is true.. It limits scalability.